### ISSUE TITLE
The screenSize property is not required on the SystemHeader storybook

### ISSUE THREAD

**Issue by ydg** [VilledeMontreal/angular-ui#172]
The 'Viewport' option of the 'Canvas' view is made for that
![image](https://github.com/VilledeMontreal/angular-ui/assets/1163185/3a53581d-2a5b-4301-bbd6-60f6a73c4a3e)

**Comment by OlivierAlbertini** [comment]
That's true ! Thanks !

**Comment by OlivierAlbertini** [comment]
Hi @ydg 

Do you have the bandwidth to do it ? I can assign you if so. Let me know !

**Comment by MaudeLaflamme** [comment]
@OlivierAlbertini 
Salut @ydg ! 
I added the _screenSize_ property in order to showcase the different buttons' configuration (depending on the type of screen it's being viewed on) without having to add script to the storybook example. The reason being that I haven't managed to find how to insert script within storybook examples. So with the _screenSize_ property, I can directly add my conditional template inside <ng-template> tags IN the storybook example template, therefore allowing future users of the SystemHeader component to have the complete template in order to have a responsive component.

Hope this is clear, let me know if you have any questions (or suggestions) :)

**Comment by OlivierAlbertini** [comment]
> @OlivierAlbertini Salut @ydg ! I added the _screenSize_ property in order to showcase the different buttons' configuration (depending on the type of screen it's being viewed on) without having to add script to the storybook example. The reason being that I haven't managed to find how to insert scripts within storybook examples. So with the _screenSize_ property, I can directly add my conditional template inside ng-template tags IN the storybook example template, therefore allowing future users of the SystemHeader component to have the complete template in order to have a responsive component.
> 
> Hope this is clear, let me know if you have any questions (or suggestions) :)

Perhaps it's just linking the functionality to the viewport. When we change the viewport, it should pass the viewport value to the component like https://varya.me/blog/stories-with-different-viewports/

**Comment by ydg** [comment]
Yes, what I meant is if the end user wants to see how the component behaves in different viewport he should do it in the Viewport view of the Canvas.
